spec/components/status_card/view_preview.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module StatusCard
+  class ViewPreview < ViewComponent::Preview
+    def single_card
+      render inline: "<div class=\"govuk-grid-row govuk-!-margin-bottom-6\"><%= render(StatusCard::View.new(card))%></div>", locals: { card: card }
+    end
+
+    def multiple_cards
+      current_user = User.new(first_name: "Darth", last_name: "Vader", email: "darth@email.com", system_admin: true)
+      render partial: "pages/badges.html.erb", locals: { current_user: current_user }
```

Comment:
Should view previews only be concerned with testing the individual component itself rather than a whole partial? Did you have a reason for wanting to test `multiple_cards` specifically?

Reply:
I've removed, I thought it would look good to see several rendered, but as you said it would include the markup in the partial not the component only.